instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -90,9 +90,19 @@ defmodule OpentelemetryEcto do
         _ -> type
       end
 
+    db_system =
+      case repo.__adapter__() do
+        Ecto.Adapters.Postgres -> :postgresql
+        Ecto.Adapters.MyXQL -> :mysql
+        Ecto.Adapters.SQLite3 -> :sqlite
+        Ecto.Adapters.Tds -> :mssql
+        _ -> :other_sql
```

Comment:
This isn't guaranteed, right? A Repo could be any storage?

Reply:
Yes, true. Maybe better falling back to `nil` then?

Reply:
What about default to `:other_ecto`? This should avoid a `nil` on a required attribute.

Reply:
Yeah, makes sense to me. And if there’s no value we should just no send the property at all, right? Rather than a `nil`?

Reply:
The `ecto` (or `other_ecto`) instead of nil, given it's a required field is too much a stretch?  What do you think, @tsloughter?

Reply:
Shall we just drop this attribute if it doesn't match right now? For 99% of users that gets this attribute in and working, and we can revisit the fallback in the future?

Reply:
Yea, probably the best way forward.